### PR TITLE
[VEN-2501][VEN-2506]: PSR configuration on Arbitrum

### PR DESCRIPTION
**Pull request by Debugger022**
## Description

Resolves #VEN-2501
This PR includes VIP for **PSR** configuration on **Arbitrum One** and **Arbitrum Sepolia**.

### CODE REVIEW

**Review by GitGuru7 — APPROVED** [review]

**Review by chechu — APPROVED** [review]

**Comment by Debugger022** [comment]
Multisig tx Arbitrum Sepolia: https://sepolia.arbiscan.io/tx/0x3791df9338544edb11b2340bd3ee2fe807e08ac196f415e28454ed79ea22e7d0

**Review by truptiQL — APPROVED** [review]

File: multisig/proposals/arbitrumsepolia/vip-005/index.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import { makeProposal } from "../../../../src/utils";
+
+const ACM = "0xa36AD96441cB931D8dFEAAaC97D3FaB4B39E590F";
```
Comment (GitGuru7) [inline multisig/proposals/arbitrumsepolia/vip-005/index.ts:3]: Should maintain consistency 
```suggestion
export const ACM = "0xa36AD96441cB931D8dFEAAaC97D3FaB4B39E590F";
```

Comment (Debugger022) [reply multisig/proposals/arbitrumsepolia/vip-005/index.ts]: Got it.
Done https://github.com/VenusProtocol/vips/pull/251/commits/7b37bc688ac4e69a764173a41718c816e34b2be4

**Review by chechu — APPROVED** [review]

**Review by GitGuru7 — APPROVED** [review]

**Comment by chechu** [comment]
https://app.venus.io/#/governance/proposal/320?chainId=56

**Pull request merged**